packages/react/focus-scope/src/FocusScope.tsx

```diff
@@ -103,7 +110,8 @@ const FocusScope = React.forwardRef((props, forwardedRef) => {
   React.useEffect(() => {
     if (container) {
       focusScopesStack.add(focusScope);
-      const PREVIOUSLY_FOCUSED_ELEMENT = document.activeElement as HTMLElement | null;
+      const activeElement = document.activeElement as HTMLElement | null;
+      const previouslyFocusedElement = unmountAutoFocusElement || activeElement;
```

Comment:
This and the first conditional in the next block is slightly confusing to me, does `unmountAutoFocusElement` become previously focused? why wouldn't it be covered by `document.activeElement` per the previous code?

Reply:
I think it's the naming that's not really correct anymore.
Before `previouslyFocusElement` made sense because it was `activeElement` at the time of mounting.
But not that's not really true if we say it's `unmountAutoFocusElement || activeElement`.